tests/test_create_reports.py

```diff
@@ -76,8 +74,80 @@ def test_reporting():
     epochs = epoch(data, duration, interval, axis=-1)
     THR_classif = [[0, 2], [0, 3]]
     valid_labels = [True, True, False, True, True, True, True, True, True, True, True, True, True, True, True, True]
-    report = reporting(epochs, valid_labels, THR_classif, log={})
-    npt.assert_equal(report["labels"][0], [False, False, False, False, True, False, True, False, True, False, False,
-                                           False, False, False, False, False])
-    npt.assert_equal(report["reports"][0]['Total number of burst'], 3)
-    npt.assert_equal(report["reports"][0]['Mean duration of phasic episode'], 1.25)
+    report = reporting(epochs, valid_labels, THR_classif, time_interval=0.25, delim=3, log={})
+    classif_expected = [False, False, False, False, True, False, True, False, True, False, False,
+                                           False, False, False, False, False]
+    tmp = [i for k, i in enumerate(classif_expected) if valid_labels[k]]  # remove invalid
+    report_expected = generate_bruxism_report(tmp, 0.25, 3)
+    npt.assert_equal(report["labels"][0], classif_expected)
+    npt.assert_equal(report["reports"][0], report_expected)
```

Comment:
instead of redoing pytest for results of `generate_bruxism_reporting`, I just call the function on the expected classif. 

Please understand and remember this pytest trick ^^ It drastically reduce the complexity of pytest for function calling existing functions.